docker/build-container.sh

```
@@ -101,9 +101,9 @@ fetch_llama_tag() {
         fi
 
         # Extract matching tag from this page
-        local found_tag=$(echo "$response" | jq -r \
-            ".[] | select(.metadata.container.tags[]? | startswith(\"$tag_prefix\")) | .metadata.container.tags[] | select(startswith(\"$tag_prefix\"))" \
-            | sort -r | head -n1)
+        local found_tag=$(echo "$response" | jq -r --arg prefix "$tag_prefix" \
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)